CHANGES.rst

```diff
@@ -169,8 +169,8 @@ v8.0.0
 * Removed unicode filename from the package, working around
   pip #3894 and setuptools #704.
 
-7.1.0
------
+v7.1.0
```

Comment:
We don't want to change these headings unless we also update the tags in the repo, as rst.linker uses these headings to inject date stamps into the changelog. If you could write up a script that will remove the existing tags and recreate them with the 'v' in them, then we could include this change.